ferret/explainers/explanation_speech/loo_speech_explainer.py

```diff
@@ -125,7 +127,7 @@ def compute_explanation(
             scores=scores,
             explainer=self.NAME + "+" + removal_type,
             target=targets if n_labels > 1 else [targets],
-            audio_path=audio_path,
+            audio=audio,  # TODO GA: I don't know if this is something we want to keep
```

Comment:
As above, but the keyword argument's name here has been updated already, but now I'm confused: the class hasn't been modified!